fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -290,7 +290,7 @@ protected void setVaryAndPreferenceAppliedHeaders(final HttpServletResponse serv
         // add vary headers
         final List<String> varyValues = new ArrayList<>(VARY_HEADERS);
 
-        if (resource.isVersioned()) {
+        if (resource.isOriginalResource()) {
```

Comment:
won't every resource just need AcceptDatetime now?  Ie, every resource is an OriginalResource, no?

Reply:
So maybe that's what this check is to prevent - a memento getting accept date time?

Reply:
@bseeger Right, it would exclude mementos, timemaps and acls